client/src/components/Layout/NavBar.js

```diff
@@ -1,3 +1,38 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 
-export default () => <h1> NavBar </h1>;
+import { ReactComponent as Home } from './Home.svg';
+import { ReactComponent as PastBills } from './past bills.svg';
+import { ReactComponent as Profile } from './profile.svg';
+
+export default () => {
```

Comment:
assign the component to a constant name so it appears in react devtools, instead of appearing like anonymous component

Reply:
making debugging components harder